src/components/wb-playground/kyn-slider-wb/kyn-slider-wb.ts

```diff
@@ -0,0 +1,163 @@
+import { html, LitElement } from 'lit';
+import { customElement, property, query, state } from 'lit/decorators.js';
+import SliderStyles from './kyn-slider-wb.scss';
+
+/**
+ * Slider.
+ */
+@customElement('kyn-slider-wb')
```

Comment:
This component will need some special considerations since it is a form input. I'll talk a bit about form participation & validation on Friday, but you can compare to some of the existing form input components,

Reply:
i've added the aria-* and step values, as well as tweaking the min and max to get the accessibility test to pass, but definitely open to learning more about the form input considerations.